tests/fragments/test_generator.py

```diff
@@ -23,13 +23,7 @@ def template_fragment():
 
```

Comment:
i think a better approach is to add a fixture that makes a temporary directory in the [tmpdir](https://docs.pytest.org/en/stable/tmpdir.html#the-tmpdir-fixture). Tests that need it can just rely on the fixture and it will get cleaned up afterwards.

Reply:
[tmpdir factory might be better](https://docs.pytest.org/en/stable/tmpdir.html#the-tmpdir-factory-fixture)

Reply:
Makes sense. Addressed.